api/src/sockets/kots/DeploySocket.ts

```diff
@@ -131,8 +131,15 @@ export class KotsDeploySocketService {
                 kubectl_version: kotsAppSpec ? kotsAppSpec.kubectlVersion : "",
                 namespace: desiredNamespace,
                 manifests: b.toString("base64"),
+                previous_manifests: "",
               };
 
+              if (app.currentSequence) {
```

Comment:
`if app.currentSequence > 1`?

Is this code path only called when updating, or will it run for the first sequence of an app too?

Reply:
🤦‍♂ typescript if(var) checks that var is defined and !=0, so this will be skipped on the first sequence of an app